usbhost.h

```diff
@@ -429,7 +436,7 @@ int8_t MAX3421e< SPI_SS, INTR >::Init() {
         /* pin and peripheral setup */
         SPI_SS::SetDirWrite();
         SPI_SS::Set();
-        spi::init();
+		spi::init();
```

Comment:
Please undo this change.

Reply:
Weird, this didn't show properly in Notepad++. Updated